Plotly.Blazor/PlotlyChart.razor

```diff
@@ -214,8 +220,27 @@
     public async Task NewPlot()
     {
         await JsRuntime.NewPlot(objectReference);
+        
+        if (this.EmitEvents != PlotlyEventNames.None)
+        {
+            await RegisterEmitEvents();
```

Comment:
This registers when the plot is first created, or recreated with `NewPlot`. Changes to the parameter in between won't have any effect, which feels ok to me with the usage in the examples.